install.sh

```diff
@@ -155,10 +156,21 @@ install_packages( )
 
   if [[ "${build_hip_clang}" == false ]]; then
     # Installing rocm-dev installs hip-hcc, which overwrites the hip-vdi runtime
-    library_dependencies_ubuntu+=( "rocm-dev" )
-    library_dependencies_centos+=( "rocm-dev" )
-    library_dependencies_fedora+=( "rocm-dev" )
-    library_dependencies_sles+=( "rocm-dev" )
+
+    if [[ -z ${custom_rocm_dev+foo} ]]; then
+    # Install base rocm-dev package unless -v/--rocm-dev flag is passed
+      library_dependencies_ubuntu+=( "rocm-dev" )
+      library_dependencies_centos+=( "rocm-dev" )
+      library_dependencies_fedora+=( "rocm-dev" )
+      library_dependencies_sles+=( "rocm-dev" )
+
+    else
+    # Install rocm-specific rocm-dev package
+      library_dependencies_ubuntu+=( "rocm-dev-${custom_rocm_dev}" )
+      library_dependencies_centos+=( "rocm-dev-${custom_rocm_dev}" )
+      library_dependencies_fedora+=( "rocm-dev-${custom_rocm_dev}" )
+      library_dependencies_sles+=( "rocm-dev-${custom_rocm_dev}" )
+    fi
```

Comment:
```
( "rocm-dev${custom_rocm_dev+-$custom_rocm_dev}"  )
```
to get rid of the `if` statement.

Reply:
i.e.:
```
    # Install rocm-specific rocm-dev package
      library_dependencies_ubuntu+=( "rocm-dev${custom_rocm_dev+-$custom_rocm_dev}" )
      library_dependencies_centos+=( "rocm-dev${custom_rocm_dev+-$custom_rocm_dev}" )
      library_dependencies_fedora+=( "rocm-dev${custom_rocm_dev+-$custom_rocm_dev}" )
      library_dependencies_sles+=( "rocm-dev${custom_rocm_dev+-$custom_rocm_dev}" )
```

Reply:
(slight correction edit made)

Reply:
Sorry, I tried it several ways, and I can't seem to get it to work both set and unset.